src/zigSetup.ts

```diff
@@ -577,6 +577,37 @@ async function updateStatus(context: vscode.ExtensionContext): Promise<void> {
         });
 }
 
+async function getMirrors(context: vscode.ExtensionContext): Promise<vscode.Uri[]> {
+    let cached = { timestamp: 0, mirrors: "" };
+    const key = "mirror-cache";
```

Comment:
In case ZLS or any other application that could be integrated into the extension also has a mirror.
```suggestion
    const key = "zig-mirror-cache";
```